File: lab2-semester1/4.cpp
//C++ Program to Calculate Area of Right angle Triangle

#include <iostream>

using namespace std;

int main()
{
  float a, b, area;
  
    cout<<"Input the two sides(not base): "<<endl;
    cin>>a;
    cin>>b;
    
    area=(a*b)/2;
    
    cout<<"Area of Right angle Triangle = "<<area<<endl;

    system("pause");
    return 0;
}

File: lab2-semester1/4.go
// Program to Calculate Area of Right angle Triangle

package main

import "fmt"

func main() {
    var a, b, area float64
    fmt.Println("Input the two sides(not Hypotenuse)")
    fmt.Printf("a: ")
    fmt.Scanln(&a)
    fmt.Printf("b: ")
    fmt.Scanln(&b)
    
    area=(a*b)/2
    
    fmt.Println("Area: ", area)
}
